docs/apis-tools/tasklist-api-rest/tasklist-api-rest-authentication.md

```diff
@@ -139,3 +139,28 @@ A successful response includes [matching tasks](./specifications/search-tasks.ap
 ## Token expiration
 
 Access tokens expire according to the `expires_in` property of a successful authentication response. After this duration, in seconds, you must request a new access token.
+
+## Authentication via cookie (Self-Managed only)
+
+:::note
+When authenticating via cookie, note that Cross-Site Request Forgery (CSRF) protection must be disabled to allow this method of authentication. In a Camunda Self-Managed cluster, set the configuration property `camunda.tasklist.csrfPreventionEnabled` to `false`.
+:::
```

Comment:
I think this was removed from the similar docs (Operate or Camunda APIs...) because it was only needed for C8Run, which now does this by default. I can try and find that chain of conversation.